<launch>
    <param name="robot_description" command="cat $(find rr_description)/speedi/speedi.urdf"/>
    <node name="speedi_state_publisher" pkg ="robot_state_publisher" type="state_publisher"/>

    <node name="speedi_joint_updater" pkg="rr_description" type="speedi_joint_updater">
    	<param name="joint_state_uri" value="$(find rr_description)/speedi/persistent_joint_state.dat"/>
    	<param name="loaded_joint_state" command="cat $(find rr_description)/speedi/persistent_joint_state.dat"/>
    </node>
</launch>
